app/components/donut-chart.js

```diff
@@ -17,6 +17,15 @@ export default Ember.Component.extend({
     sizeChanged: Ember.observer('resizedSignal', function() {
         this.updateDonut();
     }),
+    
+    // Observe the width of the outer div (placeholder) surrounding the chart when browser size changes
+    parentWidthChanged: Ember.observer('parentWidth',function() {
+        if(this.get('parentWidth') < this.get('width')*150) { // If the outer div is narrower than the chart width
```

Comment:
I don't think we nieed this condition; if the outer div is larger, we should make the chart larger too.